boards/arm/faze/faze.dts

```diff
@@ -78,7 +78,7 @@
 
```

Comment:
Same as the previous patch. I think it should be squashed with the patch which update the driver. Else the board is broken between the two patches....

Reply:
You also need to fix the commit message. For example `Add handling for EN pin in driver`. This patch is no longer doing that.

Reply:
As it was told - there's another commit for handling EN pin, so why would I put it here? 🤔

Reply:
This patch is no longer adding support for the EN pin and reset. So the messages `Add handling for EN pin in driver` and `Replace partial manual reconfiguration of device's registers at its initialization with usage of dedicated reset functionality.` should not appear in the commit description. You can simply remove them or move them into another patch. It is up to you.